tests/mocks/fileMock.js

```diff
@@ -0,0 +1 @@
+export default '';
```

Comment:
- [ ] to make your repo clean, Kindly remove this empty file since you are not using it anywhere 🙏

Reply:
Hi @ichala , thanks for your comment. This file is used as a Mock. We're using that file in the Jest configuration inside the "packaga.json". It's needed to process the images of the application by Jest. Jest doesn't understand images, so this mock replaces images with an empty string.
We are using that file in the following lines of the package.json:
`
"jest": {
    "moduleNameMapper": {
      "\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/tests/mocks/fileMock.js",
      "\\.(css|less)$": "<rootDir>/tests/mocks/fileMock.js"
    }
`

Reply:
Well its now clear what it does with your awesome comment 💯